### PR TITLE
Course updates

### PR DESCRIPTION
**Pull request by jrm86**
Implemented course-related functions, and made an update to searchSectionTitles().

Removed getScheduleDates(), as it is already implemented elsewhere

### CODE REVIEW

File: src/db/addTermMgmt.sql [hunk 1]
```diff
@@ -346,6 +346,40 @@ alpha_GB_Instructor, alpha_GB_Student, alpha_GB_Registrar, alpha_GB_RegistrarAdm
...
+RETURNS TABLE(Number VARCHAR(8),
+              Title VARCHAR(100),
+              InstructorFullName VARCHAR(100),
+              StartDate DATE,
+              EndDate DATE
+             )
+AS
+$$
+SELECT number, title, COALESCE(getInstructorName(instructor1),' ') ||
+                      COALESCE(', ' getInstructorName(instructor2),' ') ||
```
Comment (afig) [inline src/db/addTermMgmt.sql]: There seems to be a missing concatenation operator between `', '` and `getInstructorName...` Seems to also occur elsewhere in the file.

File: src/db/addSectionMgmt.sql [hunk 2]
```diff
@@ -392,15 +279,14 @@ RETURNS TABLE(Number VARCHAR(8),
...
-
-SELECT CASE 
-   WHEN c.title = $2
-   THEN number, title, 0
-   ELSE number, title, 1
+SELECT number, title, 
+   CASE WHEN title = $2
+      THEN 0
+      ELSE 1
+   END
```
Comment (afig) [inline src/db/addSectionMgmt.sql:286]: This looks a lot better; thanks for finding out that this works too.

File: src/db/addTermMgmt.sql (hunk 1 above)
Comment (jrm86) [reply src/db/addTermMgmt.sql]: Thanks for the catch.  Fixed.

File: src/db/addSectionMgmt.sql (hunk 2 above)
Comment (jrm86) [reply src/db/addSectionMgmt.sql:286]: Yes, had some issues testing it out, and this was the result.  In either case, it's a bit easier to read.

**Review by DasilvaB — APPROVED** [review]
Looks good.

**Review by afig — APPROVED** [review]
Looks good, thanks.

**Pull request merged**